test/fixtures/use-input.tsx

```diff
@@ -1,8 +1,7 @@
-'use strict';
-const React = require('react');
-const {render, useInput, AppContext} = require('../..');
+import React from 'react';
+import {render, useInput, AppContext} from '../../src';
 
-const UserInput = ({test}) => {
+const UserInput: (props: { test: string }) => JSX.Element | null = ({test}) => {
```

Comment:
```suggestion
const UserInput: React.FC<{ test: string }> = ({test}) => {
```

Reply:
Resolved in https://github.com/vadimdemedes/ink/pull/264/commits/afc427c554aab7b4f763fdcdac7c9e3320f78d73